### PR TITLE
Remove last child as bug in firefox

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

**Comment by discodavey** [comment]
@davidcmoulton Have removed the last-child as causing the issue. Tested Chrome,Firefox Edge and IE11 and in various responsive widths and no issues.

**Comment by discodavey** [comment]
Tested Firefox versions 60-64 and all ok.

**Comment by thewilkybarkid** [comment]
Might want to look at #877.

**Comment by discodavey** [comment]
What was causing the issue in the screen grab on #877? I have removed the code and tested the browsers and not getting what you had. When did this occur as I cannot seem to replicate what you had in that ticket?

File: source/_patterns/01-molecules/navigation/login-control.mustache [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-<div class="login-control"
+<div class="login-control {{#isLoggedIn}}login-control--logged-in{{/isLoggedIn}}"
```
Comment (davidcmoulton) [inline source/_patterns/01-molecules/navigation/login-control.mustache]: Is this still needed?

File: assets/sass/patterns/molecules/site-header-nav-bar-secondary.scss [hunk 2]
```diff
@@ -29,8 +29,8 @@
   @include padding(0 12 0 0);
   text-transform: uppercase;
 
-  &:last-child {
-    align-self: self-end;
+  &.nav-secondary__item--logged-in {
+    align-self: stretch;
```
Comment (davidcmoulton) [inline assets/sass/patterns/molecules/site-header-nav-bar-secondary.scss]: This should only apply when JavaScript is available, we still want `baseline` cross axis alignment for the text displayed without JavaScript. Suggest the selector is modified to something like:

```.js &.nav-secondary__item--logged-in```

**Pull request merged**